mythril_multiboot2/src/main.rs

```diff
@@ -210,5 +210,5 @@ pub extern "C" fn kmain(multiboot_info_addr: usize) -> ! {
     map.insert(0usize, default_vm(0, 512, &mut multiboot_services));
     let map: &'static _ = Box::leak(Box::new(map));
 
-    vcpu::smp_entry_point(map)
+    vcpu::smp_entry_point(map);
```

Comment:
Given that `VCpu::smp_entry_point` returns (or rather, doesn't) `!`, I'd like to keep this as it was.

Reply:
:+1: not entirely sure why I added that